src/lib/sponsoredContent/data.json

```diff
@@ -0,0 +1,44 @@
+{
+  "cities": {
+    "new-york-ny-usa": {
+      "introText": "The BMW Art Guide is your go-to-guide to discover private collections of modern and contemporary art which are accessible to the public. Please see below for collections in New York.",
```

Comment:
We discussed using a placeholder here and substituting the current city name at runtime. But in the end I opted for repetitive and simple, vs. concise and clever. This way, each city's text can be editied independently as well. Easy enough to revisit if that turns out to be the wrong call.